app/configurator/map/map-chart-options.tsx

```diff
@@ -371,11 +375,20 @@ export const SymbolLayerSettings = memo(
     }, [geoDimensions]);
 
     const measuresOptions = useMemo(() => {
-      return metaData.measures.map((d) => ({
-        value: d.iri,
-        label: d.label,
-      }));
-    }, [metaData.measures]);
+      if (areaLayer.selectable) {
+        return metaData.measures
+          .filter((d) => d.iri !== areaLayer.measureIri)
+          .map((d) => ({
+            value: d.iri,
+            label: d.label,
+          }));
+      } else {
+        return metaData.measures.map((d) => ({
+          value: d.iri,
+          label: d.label,
+        }));
```

Comment:
I think it makes it a bit clearer the intent here if we only put the `if` on the filter.

```
const filterMeasures = areaLayer.selectable ?
   d => d.iri !== areaLayer.measureIri : d => true

return metaData.measures
          .filter(filterMeasures)
          .map((d) => ({
            value: d.iri,
            label: d.label,
          }));
```